client/src/components/FlashCards.jsx

```diff
@@ -31,6 +32,7 @@ const from = () => ({ x: 0, rot: 0, scale: 1.5, y: -1500 });
 const trans = (r, s) => `perspective(1500px) rotateX(15deg) rotateY(${r / 5}deg) rotateZ(${r}deg) scale(${s})`;
 
 const FlashCards = props => {
+  const [showQuizScoreDialog, setShowQuizScoreDialog] = useState(false);
   const { cardData, mode } = props;
   const classes = useStyles();
   const [score, setScore] = useState(0);
```

Comment:
Now that we're passing this value between components I'd argue that we should move it into Redux instead of passing it through props

Reply:
Style adjusted, feedback implemented, Redux added.